cybershell/__init__.py

```
@@ -0,0 +1,157 @@
+    
+    # Main classes exported for convenience
+    'CyberShell',
+    'PluginBase',
+    'PluginResult',
+    'UnifiedConfig',
+    'SafetyConfig',
+    'BountyConfig',
+    'ExploitationConfig',
+    'LLMConfig',
+    'LearningConfig'
+]
```

Ruff (0.12.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)